internal/kafka/command_link_configuration_update.go

```diff
@@ -58,7 +58,7 @@ func (c *linkCommand) configurationUpdate(cmd *cobra.Command, args []string) err
 		config = []string{configFile}
 	}
 
-	configMap, err := properties.GetMap(config)
+	configMap, err := properties.GetMapWithJavaPropertyParsing(config)
```

Comment:
I don't think we need a separate function here. Why wouldn't we want this bug fix in the existing function?

Reply:
There is a difference in behavior. The main one the new parser treats the first empty space as key value separator. For example, "key val" equals to "key=val". In the current parser, this is an error. Also there might be behavior that i don't know.
Therefore, i want to reduce the blast radius. If things are ok with CL for sometime, we can switch others to use it.

Reply:
In the latest patch, I replaced the default parser with the new parser. Let's see how it goes. If no tests failing, we can try using it. If found regression, we can either revert the change or fix it.